### PR TITLE
Sending messages with GraphQL mutations

### PR DESCRIPTION
**Pull request by milenazuccarelli**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -24,21 +29,23 @@
...
     "react": "16.8.6",
+    "react-apollo": "2.5.8",
     "react-apollo-hooks": "0.5.0",
     "react-dom": "16.8.6",
     "react-router-dom": "5.0.1",
     "react-router-transition": "1.3.0",
     "react-scripts": "3.0.1",
     "styled-components": "4.3.2",
-    "typescript": "3.5.3"
+    "typescript": "3.6.0"
```
Comment (darahayes) [inline client/package.json]: Could you revert this change? I see the build failing as a version `3.6.0` could not be found.

**Comment by MStokluska** [comment]
I think it's failing as code generated file types has been added to .git ignore and is no longer generated.
Can this be fixed by adding 
```
yarn codegen
```
to CircleCI or 
```
codegen: gql-gen 
```
to package.json

**Comment by darahayes** [comment]
> I think this can this be fixed by adding

This is exactly it. The client app now has one additional step before you can build it. You need to run the codegen. This needs to be added to the CircleCI config.

@milenazuccarelli can you add that? In the CircleCI config there is a `build_client` step that probably runs a couple of commands. Could you please add another command to run `yarn codegen` right before `yarn build`?

**Comment by milenazuccarelli** [comment]
> > I think this can this be fixed by adding
> 
> This is exactly it. The client app now has one additional step before you can build it. You need to run the codegen. This needs to be added to the CircleCI config.
> 
> @milenazuccarelli can you add that? In the CircleCI config there is a `build_client` step that probably runs a couple of commands. Could you please add another command to run `yarn codegen` right before `yarn build`?

Thanks, guys! Will do that asap

**Comment by darahayes** [comment]
Great work @milenazuccarelli especially with figuring out the various issues causing the build to break.

**Pull request merged**